resources/quality/goofoo/tpu/goofoo_near_0.40_tpu_87a_fine.inst.cfg

```diff
@@ -0,0 +1,17 @@
+[general]
+version = 4
+name = Fine Quality
+definition = goofoo_near
+
+[metadata]
+setting_version = 16
+type = quality
+quality_type = fine
+material = goofoo_tpu_87a
+variant = 0.6mm Nozzle
```

Comment:
The name of the file is inconsistent with the variant used. Please rename the file to `resources/quality/goofoo/tpu/goofoo_near_0.60_tpu_87a_fine.inst.cfg`.